internal/sbi/processor/processor.go

```diff
@@ -0,0 +1,25 @@
+package processor
+
+import (
+	"github.com/free5gc/pcf/internal/sbi/consumer"
+	"github.com/free5gc/pcf/pkg/app"
+)
+
+type Processor struct {
+	app.App
+
+	consumer *consumer.Consumer
+}
+
+func NewProcessor(pcf app.App, consumer *consumer.Consumer) (*Processor, error) {
```

Comment:
Please refer to https://github.com/free5gc/amf/pull/132/files

You don't need to implement the Consumer getter here.